.env

```diff
@@ -56,3 +56,8 @@ CCD_LOGSTASH_SERVICES=all
 #ENABLE_PSEUDO_ROLE_ASSIGNMENTS_GENERATION=true
 #ENABLE_PSEUDO_ACCESS_PROFILES_GENERATION=true
 #ROLE_ASSIGNMENT_URL=http://ccd-test-stubs-service:5555
+
+
+#ccd-next-hearing-date-updater
+CCD_NEXT_HEARING_DATE_PASSWORD=Pa55word11
```

Comment:
CCD_NEXT_HEARING_DATE_UPDATER_USER_PWD ??

Reply:
I could change it but this would lead to change the original HMAN-320/326. All names are exactly the same from this pr.